tests/transforms/test_transforms.py

```diff
@@ -68,3 +84,43 @@ def test_identity(sample: Dict[str, Tensor]) -> None:
     tr = transforms.Identity()
     output = tr(sample)
     assert_matching(output, sample)
+
+
+def test_augmentation_sequential(batch: Dict[str, Tensor]) -> None:
+    expected = {
+        "image": torch.tensor(  # type: ignore[attr-defined]
+            [[[[3, 2, 1], [6, 5, 4], [9, 8, 7]]]],
+            dtype=torch.float,  # type: ignore[attr-defined]
+        ),
+        "mask": torch.tensor(  # type: ignore[attr-defined]
+            [[[[1, 0, 0], [1, 1, 0], [1, 1, 1]]]],
+            dtype=torch.long,  # type: ignore[attr-defined]
+        ),
+        "labels": torch.tensor([[0, 1]]),  # type: ignore[attr-defined]
+    }
+    augs = transforms.AugmentationSequential(
+        K.RandomHorizontalFlip(p=1.0),
+        data_keys=["image", "mask"],
+    )
+    output = augs(batch)
+    assert_matching(output, expected)
+
+
+def test_augmentation_sequential_image_only(batch: Dict[str, Tensor]) -> None:
```

Comment:
We need tests that use this for:

1. Only torchgeo transforms
2. Only Kornia transforms
3. Torchgeo + Kornia transforms

Reply:
I added more tests for the above as well as testing using a grayscale, rgb, and multispectral images. Let me know if you want any more.